tests/test_contrib/test_missing/test_dispersion.py

```diff
@@ -107,18 +97,11 @@ def test_missingvaluesdispersion_with_numpy_with_y_targets(self):
         """
         Integration test of visualizer with numpy with y targets
         """
-        X, y = make_classification(
-            n_samples=400, n_features=20, n_informative=8, n_redundant=8,
-            n_classes=2, n_clusters_per_class=4, random_state=852
-        )
-
-        # add nan values to a range of values in the matrix
-        X[X > 1.5] = np.nan
 
         features = [str(n) for n in range(20)]
         classes = ['Class A', 'Class B']
         viz = MissingValuesDispersion(features=features, classes=classes)
-        viz.fit(X, y=y)
+        viz.fit(self.missingdata.X, y=self.missingdata.y)
         viz.poof()
 
         self.assert_images_similar(viz, tol=self.tol)
```

Comment:
Awesome clean up of the tests - the fixture definitely made the tests much simpler to read!